app/models/group.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+# OpenSCAP group
+class Group < ApplicationRecord
+  has_ancestry(primary_key_format: /\A[\w\-]+(\/[\w\-]+)*\z/)
```

Comment:
This would need some explanation :exploding_head: :thinking:

Reply:
When I was trying to assign a parent or parent_id, it wasn't working correctly because of how our id's are formatted so this is the solution I found online for getting it to work with uuid:
https://github.com/stefankroes/ancestry/wiki/Usage-with-UUID's

Reply:
Please put a comment above it, I might bump on this multiple times when I'm tired :see_no_evil: